app/lib/efile/nc/d400_calculator.rb

```diff
@@ -0,0 +1,38 @@
+module Efile
+  module Nc
+    class D400Calculator < ::Efile::TaxCalculator
+      attr_reader :lines
+
+      def initialize(year:, intake:, include_source: false)
+        @year = year
+        @intake = intake
+        @filing_status = intake.filing_status.to_sym
+        @direct_file_data = intake.direct_file_data
+        # intake.state_file_w2s.each do |w2|
+        #   dest_w2 = @direct_file_data.w2s[w2.w2_index]
+        #   dest_w2.node.at("W2StateLocalTaxGrp").inner_html = w2.state_tax_group_xml_node
+        # end
```

Comment:
actually, it looks like since the warshington PR, every uncommented line in this method got moved into the superclass constructor, so i deleted this whole method